ynput/ayon-core
# Core: Use older OpenColorIO

## Changelog Description
Use older version of OpenColorIO to avoid issues with crashing AYON launcher.

## Additional info
Older versions of OpenColorIO are not causing any issues.

## Testing notes:
1. Create package, upload to server add to bundle.
2. Create dependency package for the bundle.
3. Import of OCIO should not cause crash of AYON launcher (`import PyOpenColorIO`).

Resolves https://github.com/ynput/ayon-core/issues/1019
